Makefile.am

```diff
@@ -48,28 +48,24 @@ EXTRA_DIST+= COPYING
 
 # Cleanup individual files in order to preserve uninstall/etc order
 maintainer-clean-local:
-	-rm Makefile.in
-	-rm aclocal.m4
-	-rm build-aux/compile
-	-rm build-aux/config.guess
-	-rm build-aux/config.sub
-	-rm build-aux/depcomp
-	-rm build-aux/install-sh
-	-rm build-aux/ltmain.sh
-	-rm build-aux/missing
-	-rmdir build-aux
-	-rm configure
-	-rm config.log
-	-rm config.status
-	-rm config.in
-	-rm m4/libtool.m4
-	-rm m4/ltoptions.m4
-	-rm m4/ltsugar.m4
-	-rm m4/ltversion.m4
-	-rm m4/lt~obsolete.m4
-	find . -type f -name '*~' -exec rm -f '{}' \;
-	-rm -f @PACKAGE@-*.tar.gz
-	-rm -f ./pkg/rpm/@PACKAGE@-*.rpm
-	-rm -f ./pkg/deb/@PACKAGE@-*.deb
+	cd @abs_top_srcdir@; \
```

Comment:
This is `build` right ? If so why not just nuke that directory ?

Reply:
It is the `build` dir. However since it is still possible to build the library from the top src directory, i.e. `@abs_top_srcdir@`    it would cause an error to just do `rm -rf build`. Could add a check though and delete files or folders depending on `@abs_top_srcdir@ == @abs_builddir@`

Reply:
Do you think that's an use case worth supporting ?

Reply:
> This is build right ? If so why not just nuke that directory ?
 
Just nuking `build` it results in problems when doing `make maintainer-clean` since it recursively calls other make clean targets which expects files in the build dir